Framework/Geometry/src/Instrument/ComponentInfoBankHelpers.cpp

```diff
@@ -66,7 +66,7 @@ bool isSaveableBank(const ComponentInfo &compInfo, const size_t idx) {
     if (parentType != Beamline::ComponentType::Rectangular &&
         parentType != Beamline::ComponentType::Structured &&
         parentType != Beamline::ComponentType::Grid) {
-      // if component at index is not a tube then identify it as a bank
+      // check if component is assembly, if so returns false
```

Comment:
I don't understand your comment. It's completely legitimate for an assembly to be saved as a `NXdetector` isn't it?

Reply:
i think this comment was above an earlier codeblock now deleted. removed